.github/workflows/ci.yml

```diff
@@ -1,13 +1,28 @@
-name: Lint
-
 on: [push]
 
 jobs:
-  eslint:
+  js-lint:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
       - uses: stefanoeb/eslint-action@1.0.2
-
-      - name: js-unit-tests
+  js-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Build
+        run: yarn install
+      - name: Test
         run: yarn test
```

Comment:
reorganized this area a little bit too.

future: it's useful to check in a package.lock.json to pin down dependencies, and update the installation command to `yarn install --frozen-lockfile`

Reply:
ah i see, i don't know much about yarn (still learning all this js stuff), but go ahead and check that in too if you see it fit!